From 20e4b440cec1226d9c2835f54f4ae36768198166 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Tue, 15 Oct 2019 18:05:09 +0800
Subject: [PATCH] power: supply: rk817_battery: fix gcc warning

drivers/power/supply/rk817_battery.c: In function 'rk817_bat_pm_resume':
drivers/power/supply/rk817_battery.c:3290:34: warning: 'time_step' may be used uninitialized in this function [-Wmaybe-uninitialized]
      (battery->sleep_dischrg_sec > time_step)) {
      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~

Change-Id: I5ca86f00471e467448145bcb001d17b0ed242c49
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/power/supply/rk817_battery.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/power/supply/rk817_battery.c b/drivers/power/supply/rk817_battery.c
index 0b4f0088d46b..58506d0e3e77 100644
--- a/drivers/power/supply/rk817_battery.c
+++ b/drivers/power/supply/rk817_battery.c
@@ -3256,7 +3256,7 @@ static int rk817_bat_pm_resume(struct device *dev)
 {
 	struct platform_device *pdev = to_platform_device(dev);
 	struct rk817_battery_device *battery = dev_get_drvdata(&pdev->dev);
-	int interval_sec = 0, time_step, pwroff_vol;
+	int interval_sec = 0, time_step = 0, pwroff_vol;
 
 	battery->s2r = true;
 	battery->current_avg = rk817_bat_get_avg_current(battery);
-- 
2.35.3

